src/main/java/Domain/Game/Stadium.java

```diff
@@ -1,4 +1,150 @@
 package Domain.Game;
 
-public class Stadium {
+import Domain.EntityManager;
+import Domain.Users.TeamOwner;
+import Service.UIController;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class Stadium implements Asset{
+
+    private String stadName;
+    private String stadLocation;
+    private List<Team> homeTeams;
+
+    public final String namePropertyString = "Name";
+
+
+    public Stadium(String location)
+    {
+        homeTeams = new ArrayList<>();
```

Comment:
add to EntityManager or remove this constructor

Reply:
deleted!